mmv1/third_party/terraform/services/compute/resource_compute_network_edge_security_service_sweeper.go.erb

```diff
@@ -0,0 +1,129 @@
+<% autogen_exception -%>
+package compute
+<% unless version == 'ga' -%>
+
+import (
+	"context"
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+	"github.com/hashicorp/terraform-provider-google/google/sweeper"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func init() {
+	sweeper.AddTestSweepers("ComputeNetworkEdgeSecurityService", testSweepComputeNetworkEdgeSecurityService)
+}
+
+// At the time of writing, the CI only passes us-central1 as the region
+func testSweepComputeNetworkEdgeSecurityService(region string) error {
+	resourceName := "ComputeNetworkEdgeSecurityService"
+	log.Printf("[INFO][SWEEPER_LOG] Starting sweeper for %s", resourceName)
+
+	config, err := sweeper.SharedConfigForRegion(region)
+	if err != nil {
+		log.Printf("[INFO][SWEEPER_LOG] error getting shared config for region: %s", err)
+		return err
+	}
+
+	err = config.LoadAndValidate(context.Background())
+	if err != nil {
+		log.Printf("[INFO][SWEEPER_LOG] error loading: %s", err)
+		return err
+	}
+
+	t := &testing.T{}
+	billingId := envvar.GetTestBillingAccountFromEnv(t)
+
+	regions := []string{"us-central1", "us-west2", "us-south1", "southamerica-west1", "europe-west1"}
+	for _, r := range regions {
+		// Setup variables to replace in list template
```

Comment:
We can add a log here like `log.Printf("[INFO][SWEEPER_LOG] Starting sweeper for %s in %s", resourceName, r)
` ?